app/client/cli/debug.go

```diff
@@ -231,18 +231,19 @@ func sendDebugMessage(cmd *cobra.Command, debugMsg *messaging.DebugMessage) {
 		logger.Global.Error().Err(err).Msg("Failed to create Any proto")
 	}
 
-	addrBook, err := fetchAddressBook(cmd)
+	pstore, err := fetchPeerstore(cmd)
 	if err != nil {
-		logger.Global.Fatal().Msg("Unable to retrieve the addrBook")
+		logger.Global.Fatal().Msg("Unable to retrieve the pstore")
 	}
 
 	var validatorAddress []byte
-	if len(addrBook) == 0 {
+	if pstore.Size() == 0 {
 		logger.Global.Fatal().Msg("No validators found")
 	}
 
+	// DISCUSS_IN_THIS_COMMIT: what does the following comment mean?
```

Comment:
Look at `client/cli/debug.go`, specifically this code snippet:

```go
	// by default, we point at the same endpoint used by the CLI but the debug client is used either in docker-compose of K8S, therefore we cater for overriding
	validator1Endpoint := defaults.Validator1EndpointDockerCompose
	if runtime.IsProcessRunningInsideKubernetes() {
		validator1Endpoint = defaults.Validator1EndpointK8S
	}
```

When we broadcast a message:
1. Client (i.e. CLI) sends a message to one node in the network; `[0]` was chosen arbitrarily
2. Node broadcasts it to everyone else in the network; business logic for gossiping messages


DOes that make sense?